pkgs/development/python-modules/spacy-transformers/default.nix

```diff
@@ -11,11 +12,13 @@
 
 buildPythonPackage rec {
   pname = "spacy-transformers";
-  version = "1.0.2";
+  version = "1.0.6";
+
+  disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-AYshH2trMTgeSkAPRb6wRWpm4gA5FaKV2NJd+PhzAy4=";
+    sha256 = "sha256-zkpSaiqb0wUTugmbeREVJyZzv5qxXXw4YFBpXzdSUXE=";
```

Comment:
I think `hash =` is slightly preferred to `sha256 =` since `hash =` is more future-proof.

Reply:
> I think `hash =` is slightly preferred to `sha256 =` since `hash =` is more future-proof.

it does not matter

Reply:
At some point `hash =` will become the standard (There is an open PR/issue for that as far as I remember) but till then `sha256 =` can be update by `nix-update` and doesn't need manual adjustments.